src/style/badreset.css

```diff
@@ -0,0 +1,11 @@
+html {
+  font-family: "Roboto", "Segoe UI", sans-serif;
+}
+
+body,
+h2,
+p,
+div {
+  margin: 0;
+  box-sizing: border-box;
```

Comment:
box-sizing shouldn't be with reset styles, add  box-sizing to elements where you need it